docs/blocks/avoiding-properties-limit.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Avoiding Properties Limit
+category: Tutorials
+tags:
+    - guide
+    - advanced
+mentions:
+    - Kaioga5
+---
+
+## Introduction:
+As you may know, block properties have a total limit of 64 values (16 maximum per property) that you cannot exceed. This is very frustrating when creating new blocks, so I will explain you how you can avoid reaching this limit. Please note that you are not going to have more than 64 properties, however the results you'll get using this method will be equal to that!
```

Comment:
Change it to
> Block properties have a total limit of 64 values (16 maximum per property) that cannot be exceed. This article will explain how to avoid reaching the limit.
>
> :::tip
> You are not going to have more than 64 properties. Using this method will be equal to that amount!
> :::